src/envoy/http/authn/authenticator_base.cc

```diff
@@ -27,6 +26,12 @@ namespace Http {
 namespace Istio {
 namespace AuthN {
 namespace {
+// The HTTP header from which to get the verified Jwt result.
+// It is currently hard-coded. After jwt-auth has a
+// parameter for this header, the hardcoded parameter will
+// be removed.
+const std::string kJwtHeaderKey = "sec-istio-auth-userinfo";
```

Comment:
you can define LowerCaseString here, not to construct LowerCaseString everytime it is used.

Reply:
Done.

Reply:
Am I looking at the wrong version?   The type is still std::string.

Reply:
Just upload the change, please take a look.